cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectConfigurationBuilderTest.java

```diff
@@ -398,7 +402,7 @@ public void testWithRackId() {
     @ParallelTest
     public void testWithConfigProviders() {
         String configuration = new KafkaConnectConfigurationBuilder(BOOTSTRAP_SERVERS)
-                .withUserConfigurations(null)
+                .withUserConfiguration(null, true, true)
```

Comment:
This should be all false to preserve the existing behavior.

Reply:
Done